ramls/types/resources/resourceRelationships.json

```diff
@@ -20,6 +20,11 @@
       "type": "object",
       "description": "Relationship of package for a given object",
       "$ref": "../metaIncluded.json"
+    },
```

Comment:
Can we have "resources" in the "relationships" of a resource? I thought we can only have "provider", "title" or "package" as mentioned here: https://s3.amazonaws.com/foliodocs/api/mod-kb-ebsco/eholdings.html#eholdings_resources__resourceid__get

Reply:
It's because "included" contains "prpviders", "packages", "titles" and each of them requires relationships that are different. Resources are added to the scheme in order not to create separated schema for resource included parameters.
![relationships](https://user-images.githubusercontent.com/44401279/48483776-09264a00-e81d-11e8-841d-2b5a1eb2b511.PNG)